packages/grafana-ui/src/components/Combobox/useComboboxFloat.ts

```diff
@@ -22,7 +22,10 @@ export const useComboboxFloat = (items: Array<ComboboxOption<string | number>>,
   const inputRef = useRef<HTMLInputElement>(null);
   const floatingRef = useRef<HTMLDivElement>(null);
   const scrollRef = useRef<HTMLDivElement>(null);
-  const [popoverMaxSize, setPopoverMaxSize] = useState<{ width: number; height: number } | undefined>(undefined);
+  const [popoverMaxSize, setPopoverMaxSize] = useState<{ width: number; height: number }>({
+    width: 500,
+    height: 500,
```

Comment:
What do you think about just using 0?

Reply:
Interesting take, I think that makes sense.